openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -267,7 +269,30 @@ OpenAssessment.ResponseView.prototype = {
     hasPendingUploadFiles: function() {
         return this.files !== null && !this.filesUploaded;
     },
+    /**
+     Check if there is a selected file moved or deleted before uploading
+     Returns:
+     boolean: if we have deleted/moved files or not.
+     **/
+    hasAllUploadFiles: function() {
+        var errorCheckerTriggered = false,
+            allFiles = true;
```

Comment:
I don't believe either of these variables is actually necessary - `errorCheckerTriggered` isn't used, and you can avoid `allFiles` with this pattern:

```
for(...) {
    if (...) {
        return false;
    }
}
return true;
```

Reply:
I have removed `errorCheckerTriggered ` and I have changed the code according to your suggestion. Thanks